.circleci/kube.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+TIMESTAMP=$(date +%s)
+
+if [ "$GIT_BRANCH" = "master" ]; then
+  HOST=$PROD_HOST
+  echo "$PROD_CA_CERT" | base64 -i --decode > ca.pem
+  echo "$PROD_ADMIN_KEY" | base64 -i --decode > admin-key.pem
+  echo "$PROD_ADMIN_CERT" | base64 -i --decode > admin.pem
+else
+  exit 0
+fi
+docker build --rm=false -t coderdojo/events-service:"${GIT_SHA1}-${TIMESTAMP}" .
+docker login -u "$DOCKER_USER" -p "$DOCKER_PASS" -e "$DOCKER_EMAIL"
+docker push coderdojo/events-service:"${GIT_SHA1}-${TIMESTAMP}"
```

Comment:
Do we have to create a repo before pushing ?

Reply:
Good point. I'd imagine we do.

Reply:
Done https://hub.docker.com/r/coderdojo/events-service/